src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -1,46 +1,86 @@
 package cromwell.engine
 
+import java.util.UUID
+
 import akka.actor.{Actor, Props}
 import akka.event.{Logging, LoggingReceive}
 import cromwell.binding._
 import cromwell.binding.values.WdlValue
-import cromwell.engine
-import cromwell.engine.backend.Backend
+import cromwell.engine.backend.{TaskAbortedException, Backend}
 import cromwell.engine.workflow.WorkflowActor
 import cromwell.engine.workflow.WorkflowActor.CallFailed
 
 import scala.language.postfixOps
-import scala.util.{Failure, Success}
-
+import scala.util.{Try, Failure, Success}
 
 object CallActor {
+
   sealed trait CallActorMessage
-  case class Start(inputs: CallInputs) extends CallActorMessage
-  case class ExecutionFinished(outputs: CallOutputs) extends CallActorMessage
-  case class ExecutionFailed(regret: Throwable) extends CallActorMessage
+  case class Start(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backend: Backend, workflowDescriptor: WorkflowDescriptor) extends CallActorMessage
```

Comment:
I made a few changes to allow CallActors to be re-used for many calls. Most of those changes have now been reversed but in this case I still like the idea of having the Start arguments provided by the Start message rather than floating around as global variables in the CallActor